.circleci/config.yml

```diff
@@ -47,17 +300,105 @@ jobs:
       - save_cache: # saves the project dependencies
           paths:
             - ~/.m2
+          key: dockstore-cli-java-{{ checksum "pom.xml" }}
+
+  install_yq:
+    steps:
+      - run:
+          name: Install yq
+          command: |
+            wget https://github.com/mikefarah/yq/releases/download/3.3.2/yq_linux_amd64
+            chmod a+x yq_linux_amd64
+            sudo mv yq_linux_amd64 /usr/bin/yq
+
+  install-git-secrets:
+    steps:
+      - run:
+          name: Install git-secrets
+          command: |
+            wget --no-verbose -O git-secrets-1.3.0.tar.gz https://github.com/awslabs/git-secrets/archive/1.3.0.tar.gz
+            tar -zxf git-secrets-1.3.0.tar.gz
+            cd git-secrets-1.3.0
+            sudo make install
+
+  setup_postgres:
+    steps:
+      - run:
+          name: setup postgres user and DB
+          command: |
+            psql -c "create user dockstore with password 'dockstore' createdb;" -U postgres
+            psql -c 'create database webservice_test with owner = dockstore;' -U postgres
+            psql -c "ALTER USER dockstore WITH superuser;" -U postgres
+
+  install_postgres_sql_client:
+    steps:
+     - run:
+         name: Install postgresql client
```

Comment:
Think this means the postgresql client is always the latest, might have been like this before but is something to be aware of

Reply:
Added a comment in the code